labgrid-web-client/src/app/app.component.css

```diff
@@ -43,12 +43,10 @@ html, body {
 }
```

Comment:
The .material-buttons class can be removed. The styling is assigned redundantly by the mat-icon-button property.
Also make sure to remove class="material-buttons" from app.component.html.